.github/workflows/update_protos.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           && ./vcpkg install grpc \
           && ./vcpkg install upb
       - name: setup-openssl
-        run: sudo apt-get install libssl-dev
+        run: apt-get update && sudo apt-get -y --no-install-recommends install libssl-dev
```

Comment:
You don't need `sudo` for the first update?

Reply:
Don't believe so, we don't have it in the dockerfiles.